package.json

```diff
@@ -34,55 +34,55 @@
     "license": "Apache-2.0",
     "dependencies": {
         "@ledgerhq/hw-transport-node-hid": "^5.51.1",
-        "axios": "^0.21.1",
+        "axios": "^0.21.4",
         "bip32": "^1.0.2",
         "chalk": "^2.3.0",
         "cli-table3": "^0.5.1",
         "clime": "^0.5.9",
         "ora": "^3.4.0",
         "path": "^0.12.7",
-        "prompts": "^2.4.0",
-        "rxjs": "^6.6.3",
-        "symbol-hd-wallets": "0.14.1-alpha-202103052158",
+        "prompts": "^2.4.2",
+        "rxjs": "^6.6.7",
+        "symbol-hd-wallets": "0.14.2",
         "symbol-ledger-typescript": "0.0.1-alpha-202106081510",
-        "symbol-sdk": "1.0.2",
+        "symbol-sdk": "^1.0.3",
         "symbol-uri-scheme": "0.6.0",
```

Comment:
Should we wait for `uri-scheme` 0.6.1 release?

Reply:
Yes, we should, I'm going to update the PR once it's merged.